CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,51 @@
+
+## Our Standards
+
+Examples of behaviour that contributes to creating a positive environment include:
+
+- Using welcoming and inclusive language
+- Being respectful of differing viewpoints and experiences
+- Gracefully accepting constructive criticism
+- Referring to people by their preferred pronouns and using gender-neutral pronouns when uncertain
+
+Examples of unacceptable behaviour by participants include:
+
+- Trolling, insulting/derogatory comments, public or private harassment
+- Publishing others' private information, such as a physical or electronic address, without explicit permission
+- Not being respectful to reasonable communication boundaries, such as 'leave me alone,' 'go away,' or 'I’m not discussing this with you.'
+- The usage of sexualised language or imagery and unwelcome sexual attention or advances
+- Swearing, usage of strong or disturbing language
+- Demonstrating the graphics or any other content you know may be considered disturbing
+- Starting and/or participating in arguments related to politics
+- Assuming or promoting any kind of inequality including but not limited to: age, body size, disability, ethnicity, gender identity and expression, nationality and race, personal appearance, religion, or sexual identity and orientation
+- Drug promotion of any kind
+- Attacking personal tastes
+- Other conduct which you know could reasonably be considered inappropriate in a professional setting.
+
+## Our Responsibilities
+
+Project maintainers are responsible for clarifying the standards of acceptable behavior and are expected to take appropriate and fair corrective action in response to any instances of unacceptable behavior.
+
+Project maintainers have the right and responsibility to remove, edit, or reject comments, commits, code, wiki edits, issues, and other contributions that are not aligned to this Code of Conduct, or to ban temporarily or permanently any contributor for other behaviors that they deem inappropriate, threatening, offensive, or harmful.
+
+## Enforcement
+
+Violations of the Code of Conduct may be reported by sending an email to hello@gnuweeb.org. All reports will be reviewed and investigated and will result in a response that is deemed necessary and appropriate to the circumstances. Further details of specific enforcement policies may be posted separately.
+
+We hold the right and responsibility to remove comments or other contributions that are not aligned to this Code of Conduct, or to ban temporarily or permanently any members for other behaviours that they deem inappropriate, threatening, offensive, or harmful.
+
+## Attribution
+
+This Code of Conduct is adapted from [dev.to](https://dev.to "dev.to") and [Contributor Covenant](https://contributor-covenant.org/version/1/4/ "Contributor Covenant") version 1.4.
+
+[//]: # (Generated at <https://miloslav.website/code-of-conduct/>, partly copied from Contributor Covenant <https://contributor-covenant.org>.)
```

Comment:
We actually already have our own code of conduct, it's written here: https://t.me/GNUWeeb/460290

Reply:
I have made a wiki page at my forked repo and contains what's on Telegram.

Reply:
> Will this wiki page be added to original repo?
> Should I delete the CODE_OF_CONDUCT.md file?

Ah, let it be there for the moment, I may clean that up later.